### PR TITLE
Chore: throw custom Error on assertions

### PR DESCRIPTION
**Pull request by rodrigo4244**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ const makeGraceful = parser => {
 }
 
 const parsers = ({ path, graceful = true, pegOptions } = {}) => {
-  assert.strictEqual(typeof path, 'string', 'the path directory must be a string.')
-  assert.strictEqual(typeof graceful, 'boolean', 'the graceful option must be a boolean.')
+  assert.strictEqual(typeof path, 'string', Error('the path directory must be a string.'))
+  assert.strictEqual(typeof graceful, 'boolean', Error('the graceful option must be a boolean.'))
```
Comment (gunar) [inline index.js]: Why?

**Comment by gunar** [comment]
Also, rebase.

File: index.js (hunk 1 above)
Comment (rodrigo4244) [reply index.js]: Not necessary, I was testing the custom Error on node 9.

**Comment by rodrigo4244** [comment]
I'm closing this since it is not necessary.

**Pull request closed without merging**